src/main/java/com/redhat/developers/msa/aloha/AlohaVerticle.java

```diff
@@ -40,47 +44,50 @@
 import io.vertx.core.http.HttpMethod;
 import io.vertx.core.json.Json;
 import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
 import io.vertx.ext.web.handler.BodyHandler;
 import io.vertx.ext.web.handler.CorsHandler;
 import io.vertx.ext.web.handler.StaticHandler;
 
 public class AlohaVerticle extends AbstractVerticle {
+    private static final Brave BRAVE = new Brave.Builder("aloha")
+            .spanCollector(HttpSpanCollector.create(System.getenv("ZIPKIN_SERVER_URL"), new EmptySpanCollectorMetricsHandler()))
+            .build();
 
     @Override
     public void start() throws Exception {
         Router router = Router.router(vertx);
+        router.route().handler(ctx -> {
+            // note: this is *not* an example of how to properly integrate vert.x with zipkin
```

Comment:
We discussed it with @cescoffier on IRC, and there are several issues. One of them is that ZipKin kinda expects a request/response pattern, whereas this is not "enforced" by vert.x, so, there might be mismatches along the way. Another problem is that vert.x might allocate different threads for requests and responses, possibly causing problems with Brave. I don't know if there's an effort somewhere to build a ZipKin library for vert.x, but that's certainly not the goal for this task here :)